### PR TITLE
Removes hijack objective from the pool

### PR DESCRIPTION
**Pull request by AffectedArc07**
LET THE CONTROVERSY BEGIN 

## What Does This PR Do
This PR takes hijack out of the objective pool. But why AA, why would you do that?

Well, I have a long thought about this, and came to a realisation. Hijack is the least RP-centric objective we have in the entire pool. If we look at the existing objectives, most of these have atleast some RP aspect to them.

- Destroy AI: You have several routes you can go about this, either aggressively breaking into the AI sat, enslaving the AI so it kills itself, there are several ways you can do this 
- Debrain: Again, you have multiple avenues for this. You can either kill and take it that way, or offer up the person implants or another surgery then take the brain out without them knowing
- Maroon: Again, easily doable. Either kill the person, or spend a shift having a nice conversation with them to say "Heya, when the shuttle arrives could you stay here for me? Would make life a lot easier for the both of us"
- Assassinate: Again, you can go in with an ebow and a minibomb and instagib people to permakill them, or you can be subtle, let your target actually have some time to enjoy their round before being knocked out for the entire thing
- Steal: You can take the item without them noticing, drug the holder then take it when they are unconscious, trade it away, none of these involve round removal.

Now take hijack. The entire premise of this objective is mass murderbone to make sure no one gets on the shuttle. This usually results in one or more of the following:
- Dual esword rampage
- Plasma in distro
- Engine decimation
- Carpet bombing escape

Hijack just ends in complete murderbone which is boring for everyone involved except the hijacker, who usually gets owned by security and promptly shot. For the rest of the playerbase, they usually die soon off and are then out of the round for the rest of the thing. All the while this feeds into a vicious cycle of the hijacker powergaming more to be able to survive security (Entire bloodstream of memechems anyone?), and security getting hyperguns to deal with this (Backpack full of AEGs and fuck tons of cyberimps anyone?) 

I honestly think this objective discourages RP standards and promotes obscene levels of powergaming, but hey, maybe I am just crazy.

Also note that this PR just removes the ability to random roll it. Admins can still force set it. 

## Why It's Good For The Game
See entire list of points above.

## Changelog
:cl: AffectedArc07
del: You can no longer roll hijack. 
/:cl:

### CODE REVIEW

**Comment by SabreML** [comment]
I assume that the Malf AI hijack objective is still there? Since it's a pretty big part of the role.

**Comment by AffectedArc07** [comment]
> I assume that the Malf AI hijack objective is still there? Since it's a pretty big part of the role.

Yes, malf doomsday still exists, because thats the entire crew against one instead of just security against one, and the AI itself isnt as powerful as someone with veins full of memechems, and theres actual strategy to it.

**Comment by FreeStylaLT** [comment]
lslDfau
Haha, that's a good one we can now close the joke PR!

**Comment by TDSSS** [comment]
Ultimate hugboxing begins.

**Comment by warior4356** [comment]
As much as I adore my plasma fires, memechems, and toxin bombs.... I think Affected is right about this being for the best. There isn't anything interactive for the whole crew to do against a hijacker and he is correct to call it an unhealthy arms race between sec and the traitor. Overall I think this would result in better rounds overall for the crew as a whole, and reduce people being removed from a round by something they couldn't do anything against, like mass bombing or double esword slaughter. 

With this in mind, the only possible way I can think of to redeem hijack is to somehow make it crew against one like the malf AI is. Perhaps letting the traitor "declare war" on the crew. While I am offering an alternative, for me personally I am in favor of the outright removal instead of "fixing" it.

**Comment by Cocacolagua** [comment]
I personally consider hijack to be one of the best objectives from the traitor's pool for the following reasons:
- All crewmembers are impacted by it, rather than all other objectives just being the victim and security.
- The objective being a "prepare for two hours to hold this position for two minutes" allows multiple ways to go about it, you could for example:
  - Use all your time on making departures a highly-inaccesible area via building reinforced walls, sepia tiles, cannisters full of dangerous gases, breaches, you name it.
  - Use some of your time on focusing on making the place inhabitable for only twenty seconds to allow you to e-mag the shuttle, and use the rest of your time to deal with the people that managed to sneak in.
  - Good ole' fashioned murder-bone (spooky!!!)

I believe that the best of both worlds can be reached by the "declare war" suggested above, given that it lets the crew fight back rather than be at mercy of security (it also would follow the standard we have of mass-crew murdering antagonists being kill-on-sight for everyone)

**Comment by ppi13** [comment]
I agree with the removal of the objective, however I would recommend that a discussion among the heads and admins be taken, to loosen the restrictions of how 'wild' and antag can go, as say you can't set up a giant syndicate bomb next to hop, just to kill the hop. since antags is also what gives both engineering, atmospheric and medical any work to begin with.

**Comment by AffectedArc07** [comment]
> All crewmembers are impacted by it, rather than all other objectives just being the victim and security.

By impacted you mean "completely removed from the round with nothing they can do", because thats what happens. The hijacker isnt valid, the crew cant fight back in the same way they can with nukeops. They just have to go "hurr guess I am fucked then" 

> The objective being a "prepare for two hours to hold this position for two minutes" allows multiple ways to go about it, you could for example:

No one ever prepares for 2 hours. 99% of the hijack attempts are engine releases, plasmafloods, rampant rampages, and more. Theres no preparation done, its just instant death.

**Comment by BMane11** [comment]
All for this. Anytime I have witnessed a hijack performed by a robust veteran, it usually, around 8 times out of 10, result in a murder fest as AA described. There's a shortlist of regular antag players that I'm aware of that usually perform the following:

- Murder the entirety of the security force.
- Become unstoppable as a result of the typical 'powergaming' combination of tator items.
- Is not valid, so crew can't engage with the hijacker. 
- A round with a depleted or non-existing security force, dead or incapable ERT time.
- An agent who gets hijack, doesn't know what to do/overwhelmed and will either ahelp to change it or it results in an ineffective boring round for them.
- Zero RP potential in hijack, and it is exceedingly rare to see a hijacker who is not after that coveted greentext.

Now, I'm not against hijacker objective because I have seen some fun hijacks in my time, but I really believe that it should not be random and it should be doled out by the admin team when they feel the need to spice things up or are willing to trust it upon a player who can do it responsibly and in a fun method.

**Comment by TDSSS** [comment]
Okay, now for a more elaborate comment on this. First, if you're going to remove hijack, you should also remove all hijack-only traitor gear/make it nukeops only, since it won't serve much of a purpose anymore. Bye bye His Grace, I guess.

Secondly, dealing with a failing station and disasters is a valid and valuable part of SS13 gameplay. Making your way through a depressurized station, hiding out somewhere and trying to patch up your friend because medbay no longer exists, patching up power after the engine is sabotaged...all these things give people lots to do. Dealing with a disaster once things go wrong is fun. And without players to sabotage and cause said disasters, they're practically never going to happen. 

We _could_ have biohazards like terror spiders take up the slack on that, but terrors get looked at very sternly if they even think about sabotaging the engine or doing anything sabotage-adjacent.

**Comment by Wojz** [comment]
On the other side of the coin are interesting, highly involved hijacks. We'd never have Grizzle's "PENTA SM" without hijack. We'd never have Bubblegum making a disposals thing from escape to the SM chamber. We'd never have Lytic's electrical maint recycler fest. We'd never have any of the highly entertaining to watch His Grace hijacks, either. Noone will ever sabotage the engine, or blow any other large area of the station up, both of which give engineering purpose (something that department is losing rapidly as of late, nota bene).

I can't agree with this just because a large number of hijackers are shit, sorry.

**Comment by AffectedArc07** [comment]
> Okay, now for a more elaborate comment on this. First, if you're going to remove hijack, you should also remove all hijack-only traitor gear/make it nukeops only, since it won't serve much of a purpose anymore. Bye bye His Grace, I guess.

This is intentionally *not* done, because admins may wish to adminbus these things in, or adminus someone in with the objective, someone who can actually be interesting and not just "oh ahahahhaha theres plasma in distro hahahah"

> Secondly, dealing with a failing station and disasters is a valid and valuable part of SS13 gameplay. Making your way through a depressurized station, hiding out somewhere and trying to patch up your friend because medbay no longer exists, patching up power after the engine is sabotaged...all these things give people lots to do. Dealing with a disaster once things go wrong is fun. And without players to sabotage and cause said disasters, they're practically never going to happen.

Yes it is a valuable part of gameplay, but there is a difference between one crewmember doing all of this who isnt even valid, and an elite syndicate nuclear strike force, or a space wizard.

> We _could_ have biohazards like terror spiders take up the slack on that, but terrors get looked at very sternly if they even think about sabotaging the engine or doing anything sabotage-adjacent.

See above point. Nukeops, wizards, and other round types fit this niche fine.

**Comment by Cocacolagua** [comment]
Made a forum topic regarding the... improve don't remove side of the PR, so it could be discussed there instead https://www.paradisestation.org/forum/topic/20491-valid-saladitus-for-hijackers/.

**Comment by Coolrune206** [comment]
If the station still started with a Tesla/Singulo engine setup, I probably would have agreed with this PR because 90% of hijacks were just engine-loose with minimal effort from the hijacker. However, with Tesla and Singulo now far less commonly seen and thus engine-loose hijacks also reduced, I think that hijacks have been at least a little more... interesting. I find it fun both as a member of security trying to stop a hijacker and protect crew, while also, obviously, being fun to be the hijacker. Heck, I even find hijack interesting as a non-antag non-sec crewmember, just because they _usually_ bring some level of variety to the round, even if they do end up just murder-unga'ing. They have a big impact, and honestly, that's what _some_ antags should do.

**Comment by Onyxarias** [comment]
I am very in-favor for this removal.

While it is true there have been some wonderfully planned and interesting (if rare) hijacks, the vast majority of hijack players use this objective as an excuse to space half the station and kill as many people as they can.

As an MRP server, or as close to one as Para is striving to be, antags should exist to make the rounds played more interesting, and not be a way for players to prove how much of a gamer they can be by eswording half of the station. Hell, on more than one occasion I have seen a Hijack antag recall the shuttle simply so that they could keep the killing going on longer.

However, I also agree that high-energy antag objectives can be fun. While I might be for removing Hijack, perhaps some other high-intensity objectives might be in order, ones with much higher RP potential. It's something to take into consideration, I think. Regardless, removing or changing an objective like Hijack is something I could certainly see improving the server as a whole.

**Comment by trololiver112** [comment]
And while rounds with hijack isn't really that rare at all. I would go as far as to say that rounds where it all goes to shit and is really chaotic because of hijack is.

And if this now is too frequent, shouldn't we just make it more rare instead? 

Personally I don't at all mind hijack rounds at all. While i agree that some of the hijack strats are pretty lame, there is definitely some potential to make rounds interesting here as well.

**Comment by Sirryan2002** [comment]
Before objecting to this change, people should ask themselves, is the hijack objective fun for everyone? Because ultimately that is a big part of what game mechanics we remove/keep. 

It's a fact that most hijackers will resort to slaughtering people or going the quick & easy route of emagging escape and rushing the console in order to finish their objectives. This may be fun and exhilarating for them, however, in the process of hijacking, the antagonist spaces large parts of the station and kill a majority of the crew. The crew usually **cannot fight back**. Or, the entire crew is just left sitting in escape wondering "what now?" because the shuttle is gone.

Regular crew members and other antagonists become collateral to this objective. People are not able to complete objectives because they were either killed or prevented from leaving by a hijacker. People who had fun all shift can't get the satisfaction of going to CC to complete the round. 

Another issue, how can you stop a hijacker? The ultimate answer is that you really can't. Unless the hijackers have had very little preparation beforehand security can't stop them. A maxcap or plasmaflood will decimate security. You cannot prepare every time because you don't know a hijacker is coming and preparing for one is to an extent, Metagaming.

Players in this server should not feel **helpless** against a threat. They should have some opportunity to fight back. Hijacking gives an antagonist a blank check to powergame and otherwise ruins a round for other players in the name of stealing the shuttle at the end of the round. 

I agree with some of the sentiments with other people commenting, hijack round can be fun, in moderation and regulation. If not removed Hijack should be made **much rarer** and have some specific restrictions placed on what they're allowed to do. Personally, I believe this objective should be worked into something more RP based with the same intensity.  For now, I believe it is appropriate to remove it from rotation.

**Comment by MRgaty** [comment]
"B-but how do u win if u can't kill space men???"

On one hand I think having a "do whatever you want" objective is a breath of fresh air for antagonists who want to try quirky gimmicks or just click on several spess men without any OOC consequences.
On the other it is true that most hijackers tend to just HAHA CLICK ON SPESS MEN or HAHA MAXCAP BRRRBRRR and that could not be very fun for everyone.
I think I like this change tho, letting only one player/a small group of players have fun at the expense of everyone's fun is not a thing we should allow IMO

**Comment by SteelSlayer** [comment]
Personally, I enjoy the destruction and chaos hijack brings. Yes, even as a non-antag crewmember. I also don't agree that *every* hijack round is mass-murderbone and destruction, that is simply not true. And as others have mentioned this does eliminate all the interesting open-ended gimmick stuff you can do with no antag rules.

A majority of rounds which have antags in it are just snooze-fests with them quietly doing their one assasinate/steal objectives (or dying in the process) and then the rest of the crew just sits around, bored to death with nothing to do. Hijack can spice that up a bit, by adding a real threat and some danger to consider. Yeah sure you may die in the process but, come on, that's part of the game.

This will also make engineering even more boring to play than it already can be. No more hijack means no more large scale bombs, which means nothing to fix for them besides the occasional chemistry explosion, or fuel tank explosion etc. I'm not counting nuke ops and similar round types because usually no one bothers to fix anything during those rounds, and the rounds are quite short.

**Comment by datlo** [comment]
I'm conflicted on this. Some of my best memories were from hijack rounds... but usually, I'm either the hijacker or a traitor giving assistance. So the arguments make sense. And we still have wizard/nuke ops for absolute no holding back massacre.

People have been proposing to port citadel/tg's system of hijacking working by using the console instead of clearing the shuttle, but this wont solve the problem entirely. People will still murderbone.

**Comment by Spacemanspark** [comment]
Hijack is garbage and the majority of players can't be assed to try something interesting with it, made even worse by security players being forced into a "kill first think later" deal. 

If you came to Paradise with the mindset of "man murderbone is the next best thing since sliced bread" then I suggest you consider why you're choosing to play here over one of the dozen LRP servers.

Good riddance, I say.

**Comment by NotDhu** [comment]
Whether it's outright removed or the rules surrounding hijack are changed somehow, would like to see _something_ happen regarding the hijack objective.

It's currently just an excuse for antagonists to outright murderbone with no consequences. One can argue that there are other murderbone-oriented antagonists such as terrors, wizards, or nukies, but each of those antagonists gets the entire crew involved and gives everyone a chance to fight back against and interact with them. 

Hijack traitors don't do that. They present a similar level of destruction and threat to the overall crew as something like nukies or a wizard, but don't allow the crew to actually interact with or combat them at all. It just isn't fun for everyone else and doesn't get the whole crew involved the same way other murderbone-oriented antags do.

**Comment by Rurik123** [comment]
> Now take hijack. The entire premise of this objective is mass murderbone to make sure no one gets on the shuttle.

No. The entire premise of Hijack is complete freedom for the antag to do what they want, as long as it adheres to OOC rules. It is the only objective with such freedom.

> Hijack just ends in complete murderbone which is boring for everyone involved except the hijacker, who usually gets owned by security and promptly shot.

Boring for everyone? Where are you getting that from? Its boring for engineering, given something to do in repairing the breaches? Boring for security who have to catch them in a chase across the station? Boring for medbay who have to patch up civis/officers caught in the way? Unless you have actual data that says the vast majority of the server finds that boring, don't act like its a fact. 

>  All the while this feeds into a vicious cycle of the hijacker powergaming more to be able to survive security (Entire bloodstream of memechems anyone?),

Powergamers do this on any objective. The issue lies with the powergamer, not the objective.

> and security getting hyperguns to deal with this (Backpack full of AEGs and fuck tons of cyberimps anyone?)

Powergamer sec do this vs any antag, regardless of how well the antags are doing. The issue lies with the powergamer, not the objective.

> I honestly think this objective discourages RP standards and promotes obscene levels of powergaming, but hey, maybe I am just crazy.

I do agree. If we ever want to make the transition to hrp then hijack will have to go, for sure, but it doesn't promote powergaming any more than the rest of the antag objectives do. The mindset of "must win" applys to ALL objectives, and to sec as well. 

---------------------------------------------

Onto the comments.

> By impacted you mean "completely removed from the round with nothing they can do", because thats what happens. The hijacker isnt valid, the crew cant fight back in the same way they can with nukeops. They just have to go "hurr guess I am fucked then"

I want to ask for clarification here from AA. What do you mean by crew cant fight back? Half of my hijack runs that made it to the shuttle have ended in me being punched to death by a gang of civis. And rightfully so. I was trying to kill them. I've seen plenty of times poor lads had their revolver nabbed or desword stolen only to be killed by it, or crippled and left for sec. Plus cliques usually have a stunprod to ruin an unlucky murderers run, and medical have been part of the ether syringe gun fan club for years now.

If crew cant fight back I would have liked to have known that before getting fucked by them so much. Nobody told me.

> Players in this server should not feel helpless against a threat. They should have some opportunity to fight back. Hijacking gives an antagonist a blank check to powergame and otherwise ruins a round for other players in the name of stealing the shuttle at the end of the round.

If dying in a round by a man with a stechkin ruins your round then I don't know what to say. If hijacker is permakilling, powergaming with AEGs/memechems, thats them being a dick. Doubly so if their recalling the shuttle to kill more. Considering permakilling and memechem abuse happen on literally any objective that can allow killing, I don't see the issue lying with Hijack.

> Before objecting to this change, people should ask themselves, is the hijack objective fun for everyone? Because ultimately that is a big part of what game mechanics we remove/keep.

Of course its not. But neither is the stealth cling that mute stings/stunprods you in maint, drags you to a false wall, turns off your sensors, and kills you with 0 counterplay. Neither is the stealth traitor that sleepy penns you and debrains you with a surgery table at solars, or the medical doctor that takes the chance to debrain his target in medical. Or the guy that ebows his target and microwaves his brain after cutting the head open.

Yet I don't hear as much about those, which leave security with often 0 leads and just a calm, boring round of green for killer. Instead its "hur dur murderbone" as if that's the biggest offender of "boring" gameplay. Hardly. At least sec can try and combat it. Sec cant combat a murder they never knew happened. 

> If you came to Paradise with the mindset of "man murderbone is the next best thing since sliced bread" then I suggest you consider why you're choosing to play here over one of the dozen LRP servers.

I came here initially for the perfect mix of action and rp, as advertised. I stayed for the enjoyable community and the professionality of the admin team.

If enjoying action, combat which involves more than murderer and victim, and as a result enjoying both fighting and playing as hijack suddenly means I'm in the mindset "man murderbone is the next best thing since sliced bread" then so be it.

**Comment by Spacemanspark** [comment]
>I came here initially for the perfect mix of action and rp, as advertised. I stayed for the enjoyable community and the professionality of the admin team. If enjoying action, combat which involves more than murderer and victim, and as a result enjoying both fighting and playing as hijack suddenly means I'm in the mindset "man murderbone is the next best thing since sliced bread" then so be it.

I don't particularly enjoy having my statements rearranged. There is a vast difference between enjoying and interfacing with the combat mechanics of the game (something I do), and someone just taking two energy swords and going on a dull, overdone murder spree thirty minutes into a given round because coming up with something more interesting takes actual effort. 
We've seen it tens of thousands of times, and it's very... very... _very_ old. It's uninspired, it's uninteresting to many players who are actually interested and invested in actually trying to roleplay, and, frankly, I'm over it. Sooooo over it.
Removing hijack isn't going to push us to be HRP. This is merely one small issue in a massive pool of problems to solve, but an issue it still is. If you think that trying to have slightly better RP standards makes us 'HRP' then, uh... dunno what to tell you, because right now we're barely one up on most LRP server standards.

**Comment by Fox-McCloud** [comment]
Consider me strongly opposed to this.

Para is already pretty hugbox when it comes to antagging; people have an incredibly strong inclination to become upset when their round is altered from how they want to play---and this kinda only furthers that.

Do antags like this go big and loud sometimes? Sure. Can it impact the round in a huge way? You bet. It doesn't happen every round though, and it doesn't even happen every time the person has hijack.

I feel like this is trying to slowly return us to the place we were in in mid 2014 where antagging was even stricter than it was; steal objectives meant you couldn't kill the person, even if they refuse to give you a thing, and deviating even one inch would net you a bwoink----and "gimme" objectives were kinda the order of the day. I don't believe this is what we should aspire to be. Paradise is *NOT* and never has been a high RP server--we're solidly in the middle, which means, yes, sometimes you do have loads of nonsense going on, and other times it's super quiet. Having too much nonsense can, indeed, impact things negatively....but I don't personally think hijack is doing this; as pointed out by @Rurik123 (or at least implied): our players are *HYPER* meta sensitive and meta-aware of things and wiling to act on it in a split instance. Removing hijack isn't going to suddenly make security willing to have an extended dialogue with people, nor is it going to make regular antags suddenly go "oh, my ways are now corrected! I'm going to engage in an hour long hostage RP standoff!" 

Roleplay levels are the coming together of a lot of gears and moving parts---and I dare say that our *obscenely large* population is likely the largest single stress-inducer for causing our RP standards to buckle---again, hijack doesn't help with these (though our huge population will, invariably make hijack more common, I will admit--but this is a problem with population and not the objective itself).


This also fundamentally kneecaps a lot of departments in terms of what they can do with their job as an antag. A small list of things you'll never be able to touch, since these can only be done during hijack:

- Atmos shenanigans
- Toxins bombs
- Syndicate bombs (the large ones)
- Virus releases
- Poisoning your food as chef
- Overloading the supermatter
- Going delta as the AI

I know some mention that AI should still be able to do its thing...but why should it? If we're being consistent, then no one would really be able to hijack; what makes traitor AI special? 

In any event, I could list more, but I think this aptly demonstrates that there's quite a few jobs who have a lot of toys they would literally never be able to use beyond a mere hypothetical in their head. 

It also takes a way a lot of small thing you can do that aren't really...mass killing, but none the less have the chance to inconvenience a lot of random people. Stuff like booby traps, disposal loops to hell if someone picks the wrong entrance, and what have you; these aren't really killing tons of people wontonly, but they are still typically reserved for hijack traitors due to their nature to still *potentially* get a lot of people killed.



On a personal note, some of my fondest memories, as a traitor, on para were from playing hijack with other people. Neca, Jay, and I engaged in a crazy hijack involving chainsaws and atmos being on fire; it was a crazy time and something that would have never happened if this objective didn't exist.

No doubt that the objective could still "hypothetically" be handed out, but let's be honest; most players won't ask for it, and those who do, especially more than a couple of times, are just going to get labeled by admins as "dirty powergamers who love to murderbone"; it'll wind up being utilized more by players who are admins personal favorites, at that time.

For all these reasons and more, I am *strongly* opposed to the removal of hijack.

Improving RP is an admirable thing, and it's something certainly worthy of pursuing, but I don't think removing hijack is a magic bullet in this regard; not addressing the elephant in the room (massive population) and its younger sibling (administrative standards of enforcement) isn't going to improve things much, if at all......and if anything, it's just going to further entrench the "hugboxy" and extremely entitled nature that has been unfortunately nurtured on Paradise.

No thanks.

👎

**Comment by KozackWolf** [comment]
Boo! not a fan of removal! It's iconic and fun!

**Comment by Butlerblock** [comment]
While I 100% agree that hijjack should stay in the game, it does seem to be slightly more common than it should be. 

I’ve seen cling tot to rounds where a cling traitor gets hijjack ontop of  3 other traitors getting hijjack as well. This ontop of anyone ahelping to “assist” with a hijjack makes it a bit overboard and way too many people running around doing what the want 

Personally I think hijjack should be given a hardcap depending on pop and admins be less lenient in giving it away when you’re assisting a hijjacker.

**Comment by Sinestia** [comment]
I think a more useful solution in the long run would be to nerf 

Flooding is too easy for example, both from atmos and from just dumping really spicy air into distro and telling a borg to mess with vents. This one's a problem because I can't think of a simple fix for this that isn't... terrible, which means I may as well have posted nothing 🤷🏻‍♀️ . It'd also be criticized for ripping out the soul and fun of the game but imo even /tg/ needs this.

Chem bombs and TTVs are much easier, latter could do with like a 10 second delay and an obvious loud sound at least. An SM yeet button to launch it into space like some old-school SM implementations had could also be desirable if people are charge delamming it (ill admit i havent played much since new SM, so im assuming it works the same.)

 Basically, a massacre is a lot more engaging than the station itself just being rendered unusable with little effort. And good antag changes in general are those that make it reasonable to give them more freedom. Reasonable chaos gives context to engineering, medical, etc. which makes the gameplay and RP more engaging.

**Comment by AffectedArc07** [comment]
This aint going in, but it was nice to see the proportion of people who agree/disagree with this
image
I expected more opposition than this, but here we are

**Comment by TheYeetster1** [comment]
We could make hijack whitelisted, or karma locked, this would address a lot of the problems mentioned.

**Comment by Spacemanspark** [comment]
Paradise does not need to karma lock the ability to murderbone. Dear God please no.

**Comment by Samman166** [comment]
Few things, I agree that my most memorable rounds are when the station falls to utter chaos and with the lack of cloning or backed up medbay your perspective shifts from "Do my job" oriented to "Just fucking get out alive" oriented. On that subject wizards and nukies just don't have that effect.

On the topic of powergaming I think the issue is not with objectives or anything it just needs to be enforced more, even a gentle DM telling people what they are doing is kinda unfair from admins would be nice. As of now it is very dependent on the admin online if they will even care about powergame, this isn't me hating on the admin team they do lots of good work and I understand things are complicated for the very reason that things are so subjective, but the work needs to start somewhere. Also to be clear neither crew or antag are innocent in this, both "Sides" have been guilty of it a lot as of late some more than others, note that I say crew and not just sec by the way, there has been a new trend of science just carrying a meth hydrocone IV on them at all times "Just incase" I know that is nothing new but I've seen it a lot more often as of late, and we don't need to talk about botany antags do we?

And I think procdrone is on the right track with objectives like that. Actually giving them the ability to be marked complete would be hard but honestly with objectives like that I feel people can judge for themselves if it is greentext or not, if they are content just getting some friends to sign a contract instead of actually working to get it via bribes, persistence, threats, and plain old charisma (For the recruiter objective in this example) well that is their loss. This also **Might** help deal with the "Must cause redtext" mindset some people have.

Aaand finally yeah I know sloppy writing and I apologize for that but also this is all just my opinion and I understand that and so should others who disagree. We should all keep in mind that people here just want to help. This is just my two cents on the matter and some ideas and my perspective on it.
